frontend/packages/dev-console/src/components/topology/__tests__/TopologyPage.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router-dom';
+import { Tooltip } from '@patternfly/react-core';
+import { TopologyPage, renderTopology } from '../TopologyPage';
+import NamespacedPage from '../../NamespacedPage';
+import { AsyncComponent, StatusBox } from '@console/internal/components/utils';
+import ConnectedTopologyDataController from '../TopologyDataController';
+import ProjectListPage from '../../projects/ProjectListPage';
+import { topologyData } from './topology-test-data';
+import Topology from '../Topology';
+
+type TopologyPageProps = React.ComponentProps<typeof TopologyPage>;
+type RenderTopologyProps = React.ComponentProps<typeof renderTopology>;
+
+let topologyProps: TopologyPageProps;
+let renderTopologyProps: RenderTopologyProps;
+
+describe('Topology page tests', () => {
+  beforeEach(() => {
+    topologyProps = {
+      activeApplication: 'topology-test',
+      knative: false,
+      cheURL: 'che.openshift.io',
+      serviceBinding: false,
+      match: {
+        params: {
+          name: 'topology-test',
+        },
+        isExact: true,
+        path: '/graph',
```

Comment:
Interesting, I just noticed this - this is not real data... I think `path` should be `'/topology/ns/:name/graph'`

Reply:
Updated this data.